simplq/src/components/common/QueueInfo/QueueInfo.jsx

```diff
@@ -40,6 +42,7 @@ export default ({ queueId }) => {
   return (
     <div className={styles['detail']}>
       <DetailRow title="Queue status:" value={status} />
+      <DetailRow title="Queue capacity:" value={queueCapacity} large />
```

Comment:
I think it makes sense to show the slots available to the user than the queue capacity. What do you think?

Reply:
I don't have a strong opinion here, but I'll prefer total over remaining.

Reply:
Yeah, no strong opinion from my side either, thought that number of slots left is a better metric to show the user, as it might interest him more. But should be fine. 👍

Reply:
Just to weigh in here, I also share @daltonfury42's opinion about remaining slots making being more useful to users.

Reply:
@skShekhar Let's change to "Available Slots"

Reply:
Updated to available slots

Reply:
Thanks.